### ISSUE TITLE
Alt tag missing for logo image.

### ISSUE THREAD

**Issue by davemyb** [apigee/apigee-devportal-kickstart-drupal#153]
The logo in the navbar does not currently have an alt tag on it. Caught this when running https://achecker.ca for WCAG 2.0 accessibility. 
My fix is as follows:
1. In src/components/navbar/navbar-brand.twig, line 22 add "alt" after the "height" parameter: `{{ navbar_brand.image(image, width, height, alt) }}`
2. On line 35, add "alt" parameter: `{% macro image(src, width, height, alt) %}`
3. On line 36, add code to output the alt tag: `  <img src="{{ src }}" width="{{ width|default(30) }}" height="{{ height|default('auto') }}" class="mr-2" alt="{{ alt|default('Company logo') }}" />`

I guess you could leave the default value for alt off, which would result in `alt=""` on output, so screen readers should ignore the image then. 

I also added the alt parameter to block--system-branding-block.html.twig, although it's not technically necessary if you use the default value on line 36 above.

**Comment by jacine** [comment]
Thanks, yes, I think this needs to be fixed in Radix.  Will update this when the solution is decided upon.

**Comment by jacine** [comment]
A patch for this has been proposed for Radix:
https://www.drupal.org/project/radix/issues/3063109

**Comment by jacine** [comment]
The patch has been committed to Radix and will be in the next release.  Thanks!
